example/storybook/src/animations/stagger/constants.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  FigmaIcon,
+  GithubIcon,
+  TwitchIcon,
+  TwitterIcon,
+} from 'lucide-react-native';
+
+export const animationValues = [
+  {
```

Comment:
Move this configuration to component itself and make it generic. May be we can do `index * 100` or `(length - index) * 100` for delay.

Reply:
Move `:initial` and `:animate` in component itself. We don't need it in array since it is same for all icons.